packages/Path-View-Core.package/SClassDiagram.class/instance/extendDiagramAreaTo..st

```diff
@@ -0,0 +1,10 @@
+geometry
+extendDiagramAreaTo: newBounds
+
+	| bounds |
+	bounds := self boundsInWorld.
+	(bounds top > newBounds top) ifTrue: [self dragTopBy: (bounds top - newBounds top)].
```

Comment:
We could also just do the extension within the if-statement. I created methods 'dragTop', 'dragBottom', 'dragright' and 'dragleft' because we wanted to extend the diagram but keep the focus. we didn´t do that by now as it doesn´t seem necessary. So we could leave the extra functions out or just keep two of them as dragBottom and dragTop should do exactly the same currently.